<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <comment>**********************************************************************
 **
 ** Copyright Â© 2011, Jason Gedge &lt;gedge -at- ualberta -dot- ca&gt;
 **
 ** This file is part of StereoReconstruction.
 **
 ** StereoReconstruction is free software: you can redistribute it and/or
 ** modify it under the terms of the GNU General Public License as published
 ** by the Free Software Foundation, either version 3 of the License, or
 ** (at your option) any later version.
 **
 ** StereoReconstruction is distributed in the hope that it will be
 ** useful, but WITHOUT ANY WARRANTY; without even the implied warranty
 ** of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
 ** GNU General Public License for more details.
 **
 ** You should have received a copy of the GNU General Public License along
 ** with StereoReconstruction. If not, see &lt;http:www.gnu.org/licenses/&gt;.
 **
 **********************************************************************</comment>
 <class>TaskProgressWidget</class>
 <widget class="QWidget" name="TaskProgressWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>69</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>TaskProgress</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="2">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="ElidedLabel" name="labelStage">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QToolButton" name="cancelButton">
     <property name="styleSheet">
      <string notr="true">border: none;
background: none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ElidedLabel</class>
   <extends>QLabel</extends>
   <header>gui/widgets/elidedlabel.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
